components/skeleton/skeleton-group-mixin.js

```diff
@@ -0,0 +1,51 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { SkeletonMixin } from './skeleton-mixin.js';
+import { SubscriberRegistryController } from '../../controllers/subscriber/subscriberControllers.js';
+
+export const SkeletonGroupMixin = dedupeMixin(superclass => class extends SkeletonMixin(superclass) {
+
+	static get properties() {
+		// TODO: remove reflect
+		return {
+			_anySubscribersInSkeleton: { reflect: true },
+			_setByParent: { reflect: true }
+		};
+	}
+
+	constructor() {
+		super();
+		this._anySubscribersInSkeleton = false;
+		this._setByParent = false;
+		this._skeletonSubscribers = new SubscriberRegistryController(this, 'skeleton', {
+			onSubscribe: this.onSubscriberChange.bind(this),
+			onUnsubscribe: this.onSubscriberChange.bind(this),
+			updateSubscribers: this._checkSubscribersSkeletonState.bind(this),
+		});
+	}
+
+	updated(changedProperties) {
+		if (changedProperties.has('skeleton')) {
+			this._skeletonSubscribers.updateSubscribers();
+		}
+	}
+
+	onSubscriberChange() {
+		this._skeletonSubscribers.updateSubscribers();
+	}
+
+	_checkSubscribersSkeletonState(subscribers) {
+		this._anySubscribersInSkeleton = [...subscribers.values()].some(subscriber => (
+			subscriber._skeleton || subscriber._anySubscribersInSkeleton
+		));
+
+		this._skeletonActive = this._skeleton || this._anySubscribersInSkeleton || this._setByParent;
```

Comment:
It's a _little_ leaky for the group to set something that's "internal" to `SkeletonMixin`. Could it just call `setSkeletonActive()` instead?

Reply:
Addressed in c7e2d4993f7713322058d199c09e2d0ab0d04d21